files/fr/web/css/@font-face/size-adjust/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: size-adjust
+slug: Web/CSS/@font-face/size-adjust
+browser-compat: css.at-rules.font-face.size-adjust
+translation_of: 'Web/CSS/@font-face/size-adjust'
+---
+<p>{{CSSRef}}{{SeeCompatTable}}</p>
+
+<p>Le descripteur CSS <strong><code>size-adjust</code></strong> CSS définit un multiplicateur destiné aux contours des glyphes et au mesures associées à la police. Cela facilite l'harmonisation de l'apparence des polices lorsqu'elles sont rendues avec la même taille.</p>
+
+<p>Le descripteur <code>size-adjust</code> se comporte d'une manière similaire à la propriété {{cssxref("font-size-adjust")}}. Il calcule l'ajustement à effectuer par police en faisant la correspondance avec leurs hauteurs respectives.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">size-adjust: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}, avec une valeur initiale de 100%.</dd>
```

Comment:
https://github.com/mdn/translated-content/pull/1752/files#r678013156 again ^^

Reply:
Yeah, fixed in https://github.com/mdn/translated-content/pull/1755/commits/1ffa2ac870686f9ad39059e3eef0c62fce5453a6